tests/integration/utils.py

```diff
@@ -336,7 +336,7 @@ def fetch_deps_and_check_output(
     """
     Fetch dependencies for source repo and check expected output.
 
-    :param tmp_path: Temp directory for pytest
+    :param tmp_path: pytest fixture
```

Comment:
[optional][non-blocking]
s/pytest fixture/pytest fixture pointing to Cachi2 temporary directory/